Integrations/RiskSense/RiskSense.py

```diff
@@ -0,0 +1,1975 @@
+
+    :param data: prepared request
+    :param additional_fieldname: fieldname is used to create an additional filter.
+    :param exclusive_operator: exclusive_operator is used to create an additional filter.
+    :param operator: operator is used to create additional filter.
+    :param value: value is used to create additional filter.
+    :return: dict
+    """
+    filters = data.get('filters', [])
+    filtr = {'field': additional_fieldname, 'exclusive': exclusive_operator,
+             'operator': operator, 'value': value}
+    filters.append(filtr)
+    data['filters'] = filters
+
+    return data
+
+
+def get_host_context(resp_host):
+    """
+    Prepare host context data as per Demisto's standard.
+
+    :param resp_host: response from host command.
+    :return: None
+    """
+    return {
+        'ID': resp_host.get('id', ''),
+        'Hostname': resp_host.get('hostName', ''),
+        'IP': resp_host.get('ipAddress', ''),
+        'OS': resp_host.get('operatingSystemScanner', {}).get('name', '')
+    }
+
+
+def get_host_hr(resp_host):
+    """
+    Prepares human readable json for command 'risksense-get-hosts' command.
+
+    :param resp_host: response from host command.
+    :return: None
+    """
+    return {
+        'Id': resp_host.get('id', ''),
+        'Rs3': resp_host.get('rs3', ''),
+        'XRs3': resp_host.get('xRS3', ''),
+        'HostName': resp_host.get('hostName', ''),
+        'Criticality': resp_host.get('criticality', ''),
+        'IpAddress': resp_host.get('ipAddress', ''),
+        'Network': resp_host.get('network', {}).get('name', ''),
+        'Group': len(resp_host.get('groups', [])),
+        'Total Findings': resp_host.get('findingsDistribution', {}).get('total', {}).get('value', 0),
+        'Critical Findings': resp_host.get('findingsDistribution', {}).get('critical', {}).get('value', 0),
+        'High Findings': resp_host.get('findingsDistribution', {}).get('high', {}).get('value', 0),
+        'Medium Findings': resp_host.get('findingsDistribution', {}).get('medium', {}).get('value', 0),
+        'Low Findings': resp_host.get('findingsDistribution', {}).get('low', {}).get('value', 0),
+        'Info Findings': resp_host.get('findingsDistribution', {}).get('info', {}).get('value', 0),
+        'OS': resp_host.get('operatingSystemScanner', {}).get('name', ''),
+        'Tags': len(resp_host.get('tags', [])),
+        'Notes': len(resp_host.get('notes', []))
+    }
+
+
+def get_services(services):
+    """
+    Get all services from the response and make the comma-separated string.
+
+    :param services: List of services.
+    :return: comma-separated list.
+    """
+    return ', '.join(services)
+
+
+def get_risksense_host_context(href, resp_host):
+    """
+    Prepare context data under RiskSense host.
+
+    :param href:  Reference link from host resp.
+    :param resp_host: Response from host command.
+    :return: None
+    """
+    return {
+        'Id': resp_host.get('id', ''),
```

Comment:
lets change this to ID please, it's a convention, and much easier to read.
```suggestion
        'ID': resp_host.get('id', ''),
```

Reply:
@ronykoz this has been taken care